internal/fositestorage/authorizationcode/authorizationcode.go

```diff
@@ -190,6 +190,7 @@ func (e *errSerializationFailureWithCause) Error() string {
 	return fmt.Sprintf("%s: %s", fosite.ErrSerializationFailure, e.cause)
 }
 
+// TODO: need to revisit this, there is a unit test now failing.
```

Comment:
No idea why "fuzzing" is affected by these changes atm.

Reply:
New elements in the `authorizationcode.Session` will change the output of the fuzzing. It's deterministic given a particular seed so the tests are deterministic.

Reply:
Nice, this was interesting.